dist/samples/places-autocomplete-addressform/inline.html

```diff
@@ -18,84 +23,118 @@
         padding: 0;
       }
 
-      #locationField,
-      #controls {
-        position: relative;
-        width: 480px;
+      body {
+        font-family: "Roboto", sans-serif;
+        font-size: 18px;
+        color: #686868;
       }
 
-      #autocomplete {
-        position: absolute;
-        top: 0px;
-        left: 0px;
-        width: 99%;
+      form {
+        display: flex;
+        flex-wrap: wrap;
+        max-width: 400px;
+        margin: 20px auto;
+        padding: 4rem 0;
       }
 
-      .label {
-        text-align: right;
-        font-weight: bold;
-        width: 100px;
-        color: #303030;
-        font-family: "Roboto", Arial, Helvetica, sans-serif;
+      input {
+        width: 100%;
+        height: 1.2rem;
+        margin-top: 0;
+        padding: 0.5em;
+        border: 0;
+        border-bottom: 2px solid gray;
+        font-family: "Roboto", sans-serif;
```

Comment:
Agreed on keeping the "teaching" code simple! When we link to the code/embed snippets, we link to what is in [/src](https://github.com/googlemaps/js-samples/tree/master/samples/places-autocomplete-addressform/src) instead of /dist. /dist is for rendering the "working demo" that is embedded in various places like [documentation samples](https://developers.google.com/maps/documentation/javascript/examples/places-autocomplete-addressform). Within /src, the .njk and .ts files have the essential code, with styling separated out into the .scss file.